components/mpas-ocean/src/Registry.xml

```diff
@@ -691,6 +691,35 @@
 					description="The length scale of exponential decay of river runoff. Fluxes are multiplied by $e^{z/\gamma}$, where this coefficient is $\gamma$."
 					possible_values="Any positive real number."
 		/>
+		<nml_option name="config_flux_attenuation_coefficient_subglacial_runoff" type="real" default_value="0.001" units="m"
+					description="The length scale of exponential decay of subglacial runoff. Fluxes are multiplied by $e^{z/\gamma}$, where this coefficient is $\gamma$."
+					possible_values="Any positive real number."
+		/>
```

Comment:
It seems like we might need a logical config option that determines whether subglacial runoff is used at all.  It seems like that should control a package and variables should only be allocated and used if the package is on.

If we only were adding a few variables of size `nCells`, I wouldn't worry about this but `fractionAbsorbedSubglacialRunoff` also has `nVertLevels` so it's pretty big.